buyassets/simplex.json

```diff
@@ -40,6 +40,14 @@
             "android_blockchain": "MATIC_BLOCKCHAIN",
             "android_asset_protocol": "MATIC_PROTOCOL"
         },
+        {
+            "currency_code": "BNB-SC",
```

Comment:
seems like a typo

Reply:
yeah it's confusing, but actually BNB-SC is correct value
<img width="1380" alt="Screenshot 2023-06-12 at 18 05 56" src="https://github.com/bitcoin-portal/assetConvertors/assets/1723595/c8f5e3ed-716a-4904-9e42-65f04ac6b358">